WcaOnRails/app/controllers/git_hub_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class GitHubController < ApplicationController
+  def edit_translation
+  end
+
+  def update_translation
+    user_login = Octokit.user.login
+    origin_repo = "#{user_login}/worldcubeassociation.org"
+    upstream_repo = "thewca/worldcubeassociation.org"
+    content = params[:translation][:content].delete("\r") # We don't want \r characters, but browsers add them automatically.
```

Comment:
Tricky stuff. Could you check if we have any `\r` characters in our existing codebase, and if it is possible for people to add them? It looks like our [`pre_commit`](https://github.com/thewca/worldcubeassociation.org/blob/master/config/pre_commit.rb) only prevents `<<<` and `\t`.